### PR TITLE
Default attributes in control plugins

### PR DESCRIPTION
**Pull request by chandler3224**
Adds the ability to define attributes for control plugins without needing to update the formBuilder options during init. 
This also solves the following items:

#1139 
#1049

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+* text eol=lf
```
Comment (kevinchappell) [inline .gitattributes:1]:
```suggestion
* text eol=crlf
```

**Review by kevinchappell — CHANGES_REQUESTED** [review]
i really like this feature but can we have the `eol` setting be `crlf`?

File: .gitattributes (hunk 1 above)
Comment (chandler3224) [reply .gitattributes:1]: Can't see why not, though it may flag up changes on a bunch of files as these were all previously ```lf```.

**Review by kevinchappell — APPROVED** [review]
lgtm! thank you!

File: .gitattributes (hunk 1 above)
Comment (kevinchappell) [reply .gitattributes]: i think the `lf` setting is what caused these non-change changes in your pr. 
image


not a big deal but it now looks like you've made tons of changes. so many that you made the top contributors on the website :sweat_smile: 

image

Comment (chandler3224) [reply .gitattributes]: Ah :/ 
That's slightly embarrassing. I only made the change as it was breaking the build process and the majority of the other files were LF anyway. By all means do what you need to do to drop me from that list haha

Comment (kevinchappell) [reply .gitattributes]: haha. thanks for the link it buts all good.

**Pull request merged**